plugin/.eslintrc.js

```diff
@@ -9,11 +9,6 @@ module.exports = {
     },
     // Uses eslint default ruleset
     extends: "eslint:recommended",
-    plugins: [
-        // Keeps the default level to warn to avoid breaking the current
-        // CI build environment
-        "only-warn"
-    ],
```

Comment:
In recent editor changes, I inadvertently introduced new ESLint warnings but did not notice them because the build still succeeded. This turns ESLint warnings into build failures.